packages/cloud_firestore/cloud_firestore/README.md

```diff
@@ -97,6 +97,21 @@ Firestore.instance.runTransaction((Transaction tx) async {
 });
 ```
 
+## Faster iOS builds
+
+Cloud Firestore has dependencies on iOS that lead to significant build times (see [#349](https://github.com/FirebaseExtended/flutterfire/issues/349) and [https://github.com/firebase/firebase-ios-sdk/issues/4284](https://github.com/firebase/firebase-ios-sdk/issues/4284)). You can substantially reduce build times after the initial build using [incremental installation](https://medium.com/@corteggo/incremental-pod-installation-cocoapods-1-7-0-multiple-projects-302da0208af3).
+
+To use, ensure that `generate_multiple_pod_projects: true` and `incremental_installation: true` are included in the `install!` command in your `Podfile`. If there isn't one yet, you can use this:
+
+```
+install! 'cocoapods',
+   disable_input_output_paths: true,
+   generate_multiple_pod_projects: true,
+   incremental_installation: true
```

Comment:
Just happened to see this PR, so I figured I would chime in with my recent experience.

After adding `incremental_installation: true` to my project, I was running into this issue https://github.com/CocoaPods/CocoaPods/issues/8604. I believe that the post_install hook may also need to be changed to address this. 

I ended up using `generated_projects` instead of `pods_project.targets`, but I admittedly have zero experience with iOS development, so there may be a better solution.

Reply:
@aaronraff it's right

I've the Podfile setting like this

```ruby
install! 'cocoapods',
         :disable_input_output_paths => true,
         :generate_multiple_pod_projects => true,
         :incremental_installation => true

post_install do |installer|
  installer.generated_projects.each do |target|  ## Must use `installer.generated_projects` instead of `installer.pods_project.targets`
    target.build_configurations.each do |config|
      config.build_settings['ENABLE_BITCODE'] = 'NO'
    end
  end
end
```